packages/react/src/contexts/ClerkContextProvider.tsx

```
@@ -57,6 +59,9 @@ export function ClerkContextProvider(props: ClerkContextProvider) {
     factorVerificationAge,
   } = derivedState;
 
+  console.log({ session }, 'ClerkContextProvider');
+  debugger;
```

Biome (1.9.4)

[error] This is an unexpected use of the debugger statement.

Unsafe fix: Remove debugger statement


(lint/suspicious/noDebugger)